### PR TITLE
Format alg parameters using tables - QGIS File tools

### PR DESCRIPTION
**Pull request by havatv**
Goal:

Ticket(s): #4429

- [ ] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/user_manual/processing_algs/qgis/filetools.rst [hunk 1]
```diff
@@ -12,17 +12,54 @@ File tools
...
+   :widths: 20 20 20 40
+   :stub-columns: 0
+
+   * - Label
+     - Name
+     - Type
+     - Description
+   * - **File destination**
+     - ``OUTPUT``
+     - [vector: any]
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/filetools.rst]: Sure? Asking! I have no mean to test this alg.

Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/filetools.rst]:
```suggestion
     - [string]
```

File: source/docs/user_manual/processing_algs/qgis/filetools.rst [hunk 2]
```diff
@@ -12,17 +12,54 @@ File tools
...
+     - Downloaded file (from the chosen URL)
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/filetools.rst]:
```suggestion
     - The location of the downloaded file
```

File: source/docs/user_manual/processing_algs/qgis/filetools.rst (hunk 1 above)
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/filetools.rst]: Thanks for spotting this.  I had not tested properly before documenting...

File: source/docs/user_manual/processing_algs/qgis/filetools.rst [hunk 3]
```diff
@@ -12,17 +12,54 @@ File tools
...
+     - Name
+     - Type
+     - Description
+   * - **URL**
+     - ``URL``
+     - [string]
+     - The URL of the file to download.
+   * - **File destination**
+     - ``OUTPUT``
+     - [vector: any]
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/filetools.rst]: Here also

Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/filetools.rst]:
```suggestion
     - [string]
```

Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/filetools.rst]: Sorry. Should be fixed now.

**Pull request merged**